corehq/form_processor/reprocess.py

```diff
@@ -129,7 +134,8 @@ def reprocess_form(form, save=True, lock_form=True):
             form.doc_type = 'XFormInstance'
 
         cache = interface.casedb_cache(
-            domain=form.domain, lock=True, deleted_ok=True, xforms=[form]
+            domain=form.domain, lock=True, deleted_ok=True, xforms=[form],
+            load_src="reprocess",
```

Comment:
nit: make this match the form counter `reprocess_form`

Reply:
👍 2fc51193b358c3a803d227da2829dd116eb88b44